playground/app.vue

```
@@ -1,19 +1,14 @@
 <script setup lang="ts">
   import { z } from 'zod'
 
-  const schema = z.object({ fruit: z.string() })
-  const { register, getValue } = useForm({ schema, key: 'example-form' })
-  const registerValue = register('fruit')
+  const schema = z.object({ name: z.string().default('love') })
+  const { register, getValue } = useForm({ schema })
```

ESLint

[error] 'useForm' is not defined.

(no-undef)